Formula/a/abyss.rb

```diff
@@ -63,7 +63,7 @@ class Abyss < Formula
 
   def install
     python3 = "python3.12"
-    ENV.delete("HOMEBREW_SDKROOT") if MacOS.version >= :mojave && MacOS::CLT.installed?
+    ENV.delete("HOMEBREW_SDKROOT") if OS.mac? && MacOS.version >= :mojave && MacOS::CLT.installed?
```

Comment:
This can likely be deleted entirely. It seems to be from before I fixed this in brew in March 2020.